manifest/content.xml

```diff
@@ -73,4 +73,27 @@
     <button />
   </form>
 
+  <!--
+    Tabs
+      Required childr elements: options, option, tab-content, tab
+      Each option element requires a single content:text child element
+      The number of tab elements should be the same of the number of option element
+  -->
+  <tabs>
```

Comment:
iOS supports 2 or more options on a segmented control which is what we would use here. Should we suggest a soft maximum number though to ensure usability. 2 or no more than 3?